### ISSUE TITLE
Update confirmation screens with mobile alert UI

### ISSUE THREAD

**Issue by bessgreen** [department-of-veterans-affairs/va.gov-team#22723]
## Context for this task
The app contains the following variations of confirmation ("Are you sure?") screens:
- **Type 1: New page w/o mobile alert:**
     - Appointments cancelation
     - Claims file upload
- **Type 2: New page with mobile alert:**
     - SM compose/reply > Cancel message
     - SM compose/reply  > Send
- **Type 3: Same page, in-line mobile alert**
     -  Delete Contact Info 
     - Sign Out

We should standardize the UX/UI across these so it is more cohesive.

## Acceptance Criteria
1. Update the following confirmation screens in the app:
     - TBD

## Open questions
1. Should we standardize on a single UI for all confirmation screens (i.e. Type 3?) 
2. Is there a need to update copy for any of these?
3. Additional considerations for **Type 3**:
     - The UX for Type 3 consists of 2 large buttons. Currently, this Type only displays on pages that do not contain any other large buttons, aside from the one that triggers the alert
     - If we added Type 3 to the Send Message form, for example (which has 2 other large buttons), should we disable other buttons if the mobile alert is triggered? If the user hits the 'Cancel' header button, should the mobile alert display in the same spot, as if they'd selected the 'Cancel' large button on the form?

## Screenshot for reference
![Screen Shot 2021-05-26 at 3.29.43 PM.png](https://images.zenhubusercontent.com/5f64e41291069a617acc1b47/bea95ea5-5c35-4e60-a144-5272933d9e4b)

**Comment by bridgethapner** [comment]
Looping @drjecker in b/c we were talking about this very thing this morning! 

As far as priority, IMO this is nice to have but not launch critical.

If there is runway to make this update in the front-end I think we should use the yellow warning alert for "Are you sure" in all the scenarios outlined - that is what we are using in Profile for deleting personal and contact information. 

I think UX would like to do a more robust audit of how we use alerts across the app and put more thinking behind it, but that's definitely post-MVP.

**Comment by drjecker** [comment]
Agree as a nice to have. Also been working with @msmanak on some of this so our "Are you sure" patterns are more in sync. May not be perfect for launch so a UX audit post-MVP is definitely a good idea. Not only for the design patterns but also "Are you sure" copy.

**Comment by bessgreen** [comment]
@bridgethapner @drjecker thank you both! we'll keep this in the MVP backlog as a nice-to-have if we're able to get to it for app store launch. 

I've updated the ticket based on what we know (both screens will use the mobile alert CC w/ yellow tab), so all we would need here is final copy. I started workshopping some of the content in the AC above, but i'll leave this ticket tagged w/ the **Awaiting UI/UX Review** label above until the content is validated.
